### PR TITLE
Updates to address Trusty container-based beta

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: user/trusty-ci-environment.md [hunk 1]
```diff
@@ -4,233 +4,225 @@ layout: en
 permalink: /user/trusty-ci-environment/
 ---
 
-## tl;dr - Using Trusty
+## Using Trusty
 
-If you just want to get started using our Trusty beta, then add the following to your `.travis.yml`.
+In order to use Ubuntu Trusty, add the following to your `.travis.yml`.
```
Comment (cotsog) [inline user/trusty-ci-environment.md]: maybe mention sudo-enabled?

File: user/trusty-ci-environment.md [hunk 2]
```diff
@@ -4,233 +4,225 @@ layout: en
...
 
 ``` yaml
+dist: trusty
 sudo: required
+```
+
+Or, if you want to route to the container-based beta, add:
```
Comment (cotsog) [inline user/trusty-ci-environment.md]: maybe mention sudo-less?

File: user/trusty-ci-environment.md [hunk 3]
```diff
@@ -4,233 +4,225 @@ layout: en
...
+
+``` yaml
 dist: trusty
+sudo: false
+group: beta
 ```
 
 This is enabled for both public and private repositories.
```
Comment (cotsog) [inline user/trusty-ci-environment.md]: just double-checking will this be true for Trusty on container-based infra?

File: user/trusty-ci-environment.md [hunk 4]
```diff
@@ -4,233 +4,225 @@ layout: en
...
+commit push to GitHub and the start of a job on Travis CI, which works
+especially well when one's average job duration is under 3 minutes.
 
-In our Precise based environments, we've traditionally built a library
-of images based on common language runtimes like `ruby`, `go`, `php`,
-`python`, etc.
+The primary disadvantage of running with `sudo: false` is the unavailability of
+passwordless `sudo` rights.  In order to account for this disadvantage, we offer
+addons [such as `apt.packages` and
+`apt.sources`](/user/installing-dependencies/).
```
Comment (cotsog) [inline user/trusty-ci-environment.md]: can we add a warning here about the limited number of packages available on the Trusty on container beta? and what will come in the future to fix this (if applicable)?

File: user/trusty-ci-environment.md (hunk 3 above)
Comment (meatballhat) [inline user/trusty-ci-environment.md]: Yep, it's enabled

File: user/trusty-ci-environment.md [hunk 5]
```diff
@@ -4,233 +4,232 @@ layout: en
...
-If you just want to get started using our Trusty beta, then add the following to your `.travis.yml`.
+In order to use sudo-enabled Ubuntu Trusty, add the following to your
+`.travis.yml`.
 
 ``` yaml
+dist: trusty
 sudo: required
+```
+
+Or, if you want to route to the sudo-less  beta, add:
```
Comment (cotsog) [inline user/trusty-ci-environment.md]: Nit: extra whitespace between less and beta

File: user/trusty-ci-environment.md [hunk 6]
```diff
@@ -4,233 +4,232 @@ layout: en
...
-of images based on common language runtimes like `ruby`, `go`, `php`,
-`python`, etc.
+The primary disadvantage of running with `sudo: false` is the unavailability of
+passwordless `sudo` rights.  In order to account for this disadvantage, we offer
+addons [such as `apt.packages` and
+`apt.sources`](/user/installing-dependencies/).
 
-For our Trusty beta, in part due to the architecture and speed of GCE,
-we're only making two images.
+_NOTE: The allowed list of packages that may be installed varies between Precise
```
Comment (cotsog) [inline user/trusty-ci-environment.md]: ❤️

**Pull request merged**